src/main/java/org/cactoos/scalar/ItemAt.java

```diff
@@ -139,63 +142,42 @@ public ItemAt(final Iterable<T> iterable, final T fallback) {
      * @param fallback Fallback value
      */
     public ItemAt(
-        final Iterator<T> iterator,
-        final Func<Iterable<T>, T> fallback
+        final Func<Iterable<T>, T> fallback,
+        final Iterator<T> iterator
     ) {
-        this(iterator, 0, fallback);
+        this(0, fallback, iterator);
     }
 
     /**
      * Ctor.
      *
-     * @param iterable Iterable
-     * @param position Position
-     */
-    public ItemAt(final Iterable<T> iterable, final int position) {
-        this(
-            iterable.iterator(),
-            position,
-            itr -> {
-                throw new IOException(
-                    new FormattedText(
-                        "Iterator doesn't have an element at #%d position",
-                        position
-                    ).asString()
-                );
-            }
-        );
-    }
-
-    /**
-     * Ctor.
-     *
-     * @param iterator Iterator
      * @param position Position
      * @param fallback Fallback value
+     * @param iterator Iterator
      * @todo #911:30min Remove iterator constructor
      *  from org.cactoos.scalar.ItemAt.
      *  We need to avoid the usage of Iterators
      *  and replace all of their occurrences in ctors with Iterables.
      */
     public ItemAt(
```

Comment:
constructor `ItemAt (position, fallback, iterator)` needs to be covered by a unit test

Reply:
@ilyakharlamov since I also had to do #1104, I simply removed this constructor instead.